src/routes/blog/spyder-6-daniel/+page.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Spyder 6 under the hood: Editor migration, remote dev QA, test overhaul and more!"
+author:
+  - dalthviz
+  - camgerlach
```

Comment:
Shouldn't @ccordoba12 be included here? Also, I think it makes sense to put @CAM-Gerlach as first author. I see the blog post as kind of like a chronicle where CAM compiled what I have been doing around, reasons of that work prioritization and how the work was done